python/nano/src/bigdl/nano/pytorch/torch_nano.py

```diff
@@ -311,3 +311,28 @@ def run(self, *args: Any, **kwargs: Any) -> Any:
         """Only for compatibility, don't use it."""
         # this is a abstract method, so we must implement it
         pass
+
+
+def torch_nano(num_processes: int = 1,
+               use_ipex: bool = False,
+               strategy: str = "subprocess",
+               precision: Union[str, int] = 32,
+               *args, **kwargs):
+
```

Comment:
@MeouSker77 do you think we can detect the cases where users must call `self.backward()` instead of `loss.backward()`

Reply:
yes, we must call `self.backward()` instead of `loss.backward()` only when the strategy is `DeepSpeedStrategy` and has multiple models, we can detect this cases here